installer/apt/install-debian-php-sury.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+apt-get -y install \
+  apt-transport-https \
+  ca-certificates \
+  curl
+
+wget -O "/etc/apt/trusted.gpg.d/php.gpg" "https://packages.sury.org/php/apt.gpg"
+echo "deb https://packages.sury.org/php/ bullseye main" > /etc/apt/sources.list.d/php.list
+
+apt-get update
+
+# Install packages
```

Comment:
Why are you breaking this out of the standard apt package list?

Reply:
Because we need extra care not to install PHP versions such as 7.4 or 8.0, as the management of those php packages is tricky.

See #1256 

You can also find some guidance: From this anchor down https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions#php-pear

Reply:
I think this is the same for #1168, we should check it doesn't mess up everything.

Reply:
Yeah, but we can do that by specifying the versions required in the apt list can we not?

Reply:
I haven't found any other way to prevent installing php8.0 and php7.4 when installing php-pear or php-apcu.

But feel free to tweak the this part.

Reply:
I'll try again to find some better way some other time. But I have to admit this small hacky script blend perfectly in with the current install script mess :laughing:

Reply:
- `php-pear`: adding php7.3-xml is preventing any other php version to be installed. Maybe this could be added to the list.
- `php7.3-apcu`: needs to be installed with the `--no-install-recommends` flags otherwise the default php cli will be installed (7.4 or 8.0).

Reply:
I see. Then all except `php7.3-apcu` can be added to the standard list, as they all get installed as a single `apt-get install` call, or am I still not understanding?

Reply:
@paddatrapper Yes, no misunderstanding.

I prefer not to mix this logic to the huge apt package list. This feel clean to me, PHP stuff in a separate script.
But if you think we should merge everything back in the install script I can do that.

Reply:
I think, until #1246 works and is merged, we should keep things consistent and have as much in the apt-requirements file. Then only have things that require `--no-recommends` elsewhere.